src/ansys/mapdl/core/_commands/apdl/process_controls.py

```diff
@@ -22,27 +22,324 @@
+        command, and all six characters must be input.  This command is valid in any processor.
+        """
+        command = f"*ENDIF"
+        return self.run(command, **kwargs)
+
+    def elseif(
+        self,
+        val1: str = "",
+        oper1: str = "",
+        val2: str = "",
+        conj: str = "",
+        val3: str = "",
+        oper2: str = "",
+        val4: str = "",
+        **kwargs,
+    ):
+        r"""Separates an intermediate if-then-else block.
+
+        Mechanical APDL Command: `\*ELSEIF <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_ELSEIF.html>`_
+
+        Parameters
+        ----------
+        val1 : str
+            First numerical value (or parameter which evaluates to numerical value) in the conditional
+            comparison operation. ``VAL1``, ``VAL2``, ``VAL3``, and ``VAL4`` can also be character
+            strings (enclosed in quotes) or parameters for ``Oper`` = EQ and NE only.
+
+        oper1 : str
+            Operation label. A tolerance of 1.0E-10 is used for comparisons between real numbers:
+
+            * ``EQ`` - Equal (for ``VAL1`` = ``VAL2`` ).
+            * ``NE`` - Not equal (for ``VAL1`` ≠ ``VAL2`` ).
+            * ``LT`` - Less than (for ``VAL1`` < ``VAL2`` ).
+            * ``GT`` - Greater than (for ``VAL1`` > ``VAL2`` ).
+            * ``LE`` - Less than or equal (for ``VAL1``  :math:``  ``VAL2`` ).
+            * ``GE`` - Greater than or equal (for ``VAL1``  :math:``  ``VAL2`` ).
+            * ``ABLT`` - Absolute values of ``VAL1`` and ``VAL2`` before < operation.
+            * ``ABGT`` - Absolute values of ``VAL1`` and ``VAL2`` before > operation.
+
+        val2 : str
+            Second numerical value (or parameter which evaluates to numerical value) in the conditional
+            comparison operation.
+
+        conj : str
+            (Optional) Connection between two logical clauses.
+
+            * ``AND -`` - True if both clauses ( ``Oper1`` and ``Oper2`` ) are true.
+            * ``OR -`` - True if either clause is true.
+            * ``XOR -`` - True if either (but not both) clause is true.
+
+        val3 : str
+            (Optional) Third numerical value (or parameter which evaluates to numerical value).
+
+        oper2 : str
+            (Optional) Operation label. This will have the same labels as ``Oper1``, except it uses
+            ``Val3`` and ``Val4``. A tolerance of 1.0E-10 is used for comparisons between real numbers.
+
+        val4 : str
+            (Optional) Fourth numerical value (or parameter value which evaluates to a numerical value).
+
+        Notes
+        -----
+        Optional intermediate block separator within an if-then-else construct. All seven characters of the
+        command name (\*ELSEIF) must be input. This command is similar to the :ref:`if` command except that
+        the ``Base`` field is not used. The :ref:`if`, :ref:`elseif`, :ref:`else`, and :ref:`endif`
```

Comment:
`:ref:else` is also taken from the python guide...

Reply:
You might need to go for :ref:`ansys.mapdl.core.mapdl.MapdlBase.if` or similar...

Reply:
(double check the above)